docs/Tutorials/Tutorial_data/h_minus_cross_section_wbr.dat

```diff
@@ -0,0 +1,74 @@
+180.0,6.700000000000001e-20
```

Comment:
This file shouldn't really be added. I don't dislike the idea of a tutorial_data directory, but we should not duplicate data. You can point the h_minus_cross_section file to the existing data directory in the main stardis directory, and the docs already has a sun.mod file that can be pointed to as well. 

Also, I would not really call the config files "data." Rather, maybe it's better structure to have the mesa_model be its own directory with both the notebook and the .yml file needed, and similar for the parallel stardis notebook.